src/NServiceBus.AwsLambda.SQS/AwsLambdaEndpoint.cs

```diff
@@ -372,10 +372,12 @@ async Task ProcessMessageWithInMemoryRetries(Dictionary<string, string> headers,
                 // set the native message on the context for advanced usage scenario's
                 var context = new ContextBag();
                 context.Set(nativeMessage);
+                context.Set(nativeLambdaMessage);
 
                 // We add it to the transport transaction to make it available in dispatching scenario's so we copy over message attributes when moving messages to the error/audit queue
                 var transportTransaction = new TransportTransaction();
                 transportTransaction.Set(nativeMessage);
+                transportTransaction.Set(nativeLambdaMessage);
```

Comment:
Is the formatting off here or is it my github view?

Reply:
it looks ok to me?